README.md

```diff
@@ -92,15 +92,16 @@ There's a complete and functional example at the `example` folder, with more tho
-| animationCloseTime  | float         | Yes       | 300     | Time in milliseconds to execute the drawer closing animation.                           |
-| direction           | string        | Yes       | left    | Direction to open the collage, one of: ["left", "right", "top", "bottom"].              |
-| dismissWhenTouchOutside | bool      | Yes       | true    | Should the drawer be dismissed when a click is registered outside?                      |
-| fadeOpacity         | number        | Yes       | 0.6     | Opacity of the screen outside the drawer.                                               |
-| drawerScreenWidth   | number        | Yes       | 0.8     | 0 - 1, width of drawer in relation to the screen.                                       |
-| drawerScreenHeight  | number        | Yes       | 1       | 0 - 1, height of drawer in relation to the screen.                                      |                         
+| Prop                         | Type          | Optional | Default | Description                                                                                                                          |
+| ---------------------------- | ------------- | -------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| animationOpenTime            | float         | Yes      | 300     | Time in milliseconds to execute the drawer opening animation.                                                                        |
+| animationCloseTime           | float         | Yes      | 300     | Time in milliseconds to execute the drawer closing animation.                                                                        |
+| direction                    | string        | Yes      | left    | Direction to open the collage, one of: ["left", "right", "top", "bottom"].                                                           |
+| dismissWhenTouchOutside      | bool          | Yes      | true    | Should the drawer be dismissed when a click is registered outside?                                                                   |
+| fadeOpacity                  | number        | Yes      | 0.6     | Opacity of the screen outside the drawer.                                                                                            |
+| drawerScreenWidth            | number/string | Yes      | 80%     | Width of drawer on portrait orientation. Use '%' for setting the width in relation to the screen or fixed values for absolute width  |
+| drawerScreenWidthOnLandscape | number/string | Yes      | 30%     | Width of drawer on landscape orientation. Use '%' for setting the width in relation to the screen or fixed values for absolute width |
+| drawerScreenHeight           | number/string | Yes      | 100%    | Height of drawer. Use '%' for setting the width in relation to the screen or fixed values for absolute width
```

Comment:
It would be interesting to pass examples of usage on the `drawerScreenWidth`, `drawerScreenHeight` and `drawerScreenWidthOnLandscape` property.

e.g.
> Height of drawer. Pass a string containing '%' (e.g. "30%") for setting the width in relation to the screen or a number containing a fixed value from 0 to 1 (e.g. 0.3)  for absolute width

What do you think?

Reply:
Yeah, much clearer! I'll add the examples.

Reply:
Hmm, that's interesting. We have three possible values that the user might insert:
- a number between 0 and 1, and in this case it would cover percentage (0.3 is 30%)
- a string containing a percentage from 0 to 100 (e.g. "30%")
- a number bigger than 1, which in this case would be the actual value instead of a percentage, like a width of 300

Right now, we only handle the 2nd and 3rd case, maybe we could also add a condition to handle the 1st case, though this is likely for another PR.

But, for now, you are right - let's update the docs to include examples from both cases! What you said is correct.